spec/appliance_console/external_httpd_authentication_spec.rb

```diff
@@ -162,4 +163,26 @@
       subject.post_activation
     end
   end
+
+  context "#configure_ipa_http_service" do
+    before do
+      allow(subject).to receive(:say)
```

Comment:
I was thinking that `before` block is setting-up environment for executing expectation in `it' block. I am not sure what is the right way (`expect` vs `allow`) to mock some behavior  which is not tested in `it`. 
@bdunne, do you think it make sense to remove `before` and replace all `allow` with `expect` here?